tools/Ecoregionalization_workflow/BRT_model.xml

```diff
@@ -0,0 +1,110 @@
+<tool id="brt_analysis" name="BRT tool prediction" version="0.1.0+galaxy0" profile="22.05">
```

Comment:
should all IDs start with a common prefix?

Reply:
you're right!